concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+ID	NUMBER	ITALIAN	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	ITALIAN_CONCRETENESS_MEAN	ITALIAN_IMAGEABILITY_MEAN	ITALIAN_FAMILIARITY_MEAN	ITALIAN_AoA_MEAN	ITALIAN_CONTEXT_AVAILABILITY_MEAN	ITALIAN_ABSTRACTNESS_MEAN	ITALIAN_MoA_MEAN
+DellaRosa-2010-417-1	1	Abbandono	abandonment			297	300	397	374	413	455	505
+DellaRosa-2010-417-2	2	Accelerazione	acceleration			344	342	468	445	409	415	488
+DellaRosa-2010-417-3	3	Accordo	deal			262	300	468	388	368	459	547
+DellaRosa-2010-417-4	4	Affermazione	affirmation			274	265	516	424	376	375	425
+DellaRosa-2010-417-5	5	Affetto	affection	3834	LOVE (AFFECTION)	284	465	584	294	550	488	363
```

Comment:
We only have mappings of "love" glosses to this concept and "affection" glosses are unmapped. I'd therefore unmap.